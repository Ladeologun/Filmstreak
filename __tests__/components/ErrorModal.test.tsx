import renderer from 'react-test-renderer';
import {test,expect} from '@jest/globals';
import { ErrorInfoModal } from "~components";



test('renders correctly', () => {
    const tree = renderer.create(<ErrorInfoModal isModalVisible={true} handleModalDismiss={()=>{}} />).toJSON();
    expect(tree).toMatchSnapshot();
});
