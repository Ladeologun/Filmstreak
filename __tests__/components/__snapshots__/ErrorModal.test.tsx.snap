// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Modal
  animationType="fade"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={true}
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "flex": 1,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        [
          {
            "alignItems": "center",
            "backgroundColor": "#000000AA",
            "flex": 1,
            "justifyContent": "center",
          },
          undefined,
        ]
      }
    >
      <View
        style={
          [
            {
              "alignItems": "center",
              "backgroundColor": "skyblue",
              "borderRadius": 8,
              "justifyContent": "center",
              "maxHeight": 333.5,
              "paddingBottom": 10,
              "paddingHorizontal": 10,
              "paddingVertical": 7,
              "width": "65%",
            },
            undefined,
          ]
        }
      >
        <View
          style={
            {
              "alignItems": "flex-end",
              "width": "100%",
            }
          }
        >
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "opacity": 1,
              }
            }
          >
            <Image
              source={
                {
                  "testUri": "../../../src/assets/drawables/cancel.png",
                }
              }
              style={
                {
                  "height": 30,
                  "width": 30,
                }
              }
              tintColor="#0F111D"
            />
          </View>
        </View>
        <Image
          source={
            {
              "testUri": "../../../src/assets/drawables/info-icon.png",
            }
          }
          style={
            {
              "height": ""24px"",
              "width": ""24px"",
            }
          }
          tintColor="red"
        />
        <Text
          style={
            {
              "color": "black",
              "fontSize": 17,
              "fontWeight": "700",
              "lineHeight": 25,
              "marginBottom": 6,
              "marginTop": 10,
            }
          }
        >
          Error
        </Text>
        <Text
          style={
            {
              "color": "black",
              "fontSize": 15,
              "lineHeight": 25,
              "textAlign": "center",
            }
          }
        >
          An unknown error occurred
        </Text>
      </View>
    </View>
  </View>
</Modal>
`;
